test/assert.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 var test = require('tap').test;
-var assert = require('../lib/assert');
+var assert = require('../lib/assert')();
```

Comment:
Workaround to avoid modifying each test in this file. Will need to create new instance of `Assert` in every test.